DockerSetup.md

```diff
@@ -0,0 +1,78 @@
+# Local DP Dash development on Docker
+
+This document outlines the set up for the development environment using docker
+
+## Requirements
+
+- [Complete all the steps up to the initialize step](./README-PNL.md#initialize)
+
+## Prepare the Configs:
+
+- Copy the contents from the dashboard directory.
+- Create a directory called configs inside the server directory of the project.
+- Open the basePathConfig.js and replace it's contents with this.
+
+```js
+const basePathConfig = ''
+
+export default basePathConfig
+```
+
+- Comment lines 32-35. It should look like this.
+
+```js
+config.database.mong.host = 'mongodb'
+config.database.mongo.server = {}
+config.database.mongo.server.ssl = false
+// config.database.mongo.server.sslCA = ca;
+// config.database.mongo.server.sslCert = cert;
+// config.database.mongo.server.sslKey = key;
+// config.database.mongo.server.sslValidate = false;
+config.database.mongo.server.reconnectTries = 1
+config.database.mongo.server.useNewUrlParser = true
+```
+
+# Create Admin Users
+
+[The init.sh script](./README-PNL.md#initialize) outputs a dashboard directory. The directory should be in this path ${state}/dpdash/configs/dashboard.
```

Comment:
this links to the initialize section of the readme. Is that intentional? It is referencing the `init.sh` script which is a little confusing.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/358#issuecomment-1669999988